utilities_general.py

```
@@ -1,3 +1,1553 @@
-size 69953
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Feb  1 13:52:40 2022
+
+@author: labinfo
+"""
+#codereview
+try:
+    import webview
+except:
+    pass
```

Ruff

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)